gitops/snapshot.go

```diff
@@ -841,3 +842,29 @@ func CopySnapshotLabelsAndAnnotation(application *applicationapiv1alpha1.Applica
 	_ = metadata.CopyAnnotationsByPrefix(source, &snapshot.ObjectMeta, prefix)
 
 }
+
+// SetScenarioIntegrationStatusAsInvalid sets the IntegrationTestScenarioValid status condition for the Scenario to invalid.
+func SetScenarioIntegrationStatusAsInvalid(scenario *v1beta1.IntegrationTestScenario, message string) {
```

Comment:
While all of the functions in this file look good to me code-wise, should we place them somewhere else since they do not relate to Snapshots?
Maybe [helpers/integration.go](https://github.com/redhat-appstudio/integration-service/blob/main/helpers/integration.go) or similar?
Alternatively we could even create a new `helpers/scenario.go` for them?

Reply:
Yes. I thought about this, but noticed [IntegrationTestScenarioValid](https://github.com/redhat-appstudio/integration-service/blob/main/gitops/snapshot.go#L144) is in this file, then let me move out of them together.